app/controllers/api/metric_rollups_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  class MetricRollupsController < BaseController
+    REQUIRED_PARAMS = %w(resource_type capture_interval start_date).freeze
```

Comment:
@abellotti @gtanzillo should we also require `resource_ids`? Not sure if we talked about it, but limiting it to a subset of records may be best.